packages/soci-snapshotter/soci-snapshotter.spec

```
@@ -0,0 +1,53 @@
+%global gorepo soci-snapshotter
+%global gover 0.5.0
+%global rpmver %{gover}
+
+Name: %{_cross_os}soci-snapshotter
+Version: %{gover}
+Release: 1%{?dist}
+Summary: A containerd snapshotter plugin which enables lazy loading for OCI images.
+License: Apache-2.0
+URL: https://github.com/awslabs/soci-snapshotter
+Source0: https://github.com/awslabs/soci-snapshotter/archive/refs/tags/v0.5.0.tar.gz
+Source1: bundled-v%{gover}.tar.gz
+Source2: bundled-cmd.tar.gz
+Source101: soci-snapshotter.service
+Source1000: clarify.toml
+
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}libz-devel
+
+%description
+%{summary}.
```

boilerplate for FIPS:
```suggestion
%description
%{summary}.

%package bin
Summary: SOCI snapshotter binaries
Provides: %{name}(binaries)
Requires: (%{_cross_os}image-feature(no-fips) and %{name})
Conflicts: (%{_cross_os}image-feature(fips) or %{name}-fips-bin)

%description bin
%{summary}.

%package fips-bin
Summary: SOCI snapshotter binaries, FIPS edition
Provides: %{name}(binaries)
Requires: (%{_cross_os}image-feature(fips) and %{name})
Conflicts: (%{_cross_os}image-feature(no-fips) or %{name}-bin)

%description fips-bin
%{summary}.
```